### PR TITLE
check that schedule exists before attempting to use it

### PR DESCRIPTION
**Pull request by asmacdo**
https://pulp.plan.io/issues/1244

### CODE REVIEW

**Comment by dkliban** [comment]
ok test

**Comment by rbarlow** [comment]
You need to make a unit test for this.

File: server/pulp/server/async/tasks.py [hunk 1]
```diff
@@ -407,7 +407,7 @@ def on_success(self, retval, task_id, args, kwargs):
         :param kwargs:  Original keyword arguments for the executed task.
         """
         _logger.debug("Task successful : [%s]" % task_id)
-        if 'scheduled_call_id' in kwargs:
+        if 'scheduled_call_id' in kwargs and kwargs['scheduled_call_id'] is not None:
```
Comment (ipanova) [inline server/pulp/server/async/tasks.py]: you can substitute with:
```
if kwargs.get('scheduled_call_id') is not None:
```

**Comment by asmacdo** [comment]
@rbarlow (or anyone) ready for re-review

**Pull request merged**